docker-compose.yml

```diff
@@ -1,13 +1,14 @@
-version: '3'
+version: '3.4'
 services:
-  tests: &tests-config
+  release:
     build: .
     image: keboola/manage-api-tests
-    tty: true
-    stdin_open: true
+    command: composer ci
```

Comment:
Tady jsem ten command přidal, pokud by si to někdo chtěl spustit, ale myslím, že by tam ani být nemusel.